docs/ipdk-tofino.md

```diff
@@ -0,0 +1,115 @@
+IPDK on tofino
+===============
+
+# Overview
+This document describes how to build and run IPDK on Tofino.
+
+# Requirements
+- Docker (Ubuntu 20.04 base image)
+- Intel P4Studio
+- Git
+
+# Setup
+The build and execution of infrap4d uses docker.
+
+## Docker
+
+### Pull the Ubuntu 20.04 docker base image
+> docker pull ubuntu:20.04
+
+### Create container
+> docker run --privileged --cap-add=ALL -it --name infrap4d --network host -d ubuntu:20.04 /bin/bash
+
+## Git
+
+### Clone IPDK networking-recipe
+> git clone --recursive git@github.com:ipdk-io/networking-recipe.git infrap4d
+
+## Intel P4Studio
+Get the Intel P4Studio SDE version 9.11. The untarred directory is henceforth referred to as **sde**. If using an Intel Tofino Reference platform, please also download the BSP package.
+
+> tar xvf bf-sde-9.11.0-cpr.tgz
+
+# Build
+Build is done within the infrap4d docker container created above.
+
+> docker exec -it infrap4d bash
+
+### Install basic utilities
+```
+apt-get update
+apt-get install sudo git cmake autoconf gcc g++ libtool python3 python3-dev python3-distutils
+```
+
+### Build and install infrap4d dependencies
+```
+cd infrap4d/setup
+cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/
+make
+ldconfig
+```
+
+### Build Intel P4Studio SDE
+The below steps are minimal. For a more detailed installation procedure, please refer to the Intel P4Studio SDE Installation Guide
+```
+cd sde/p4studio
+./p4studio profile apply profiles/ipdk.yaml --bsp-path **path-to-bsp**
+cd ..
+export SDE_INSTALL=sde/install
+```
+
+### Build infrap4d
```

Comment:
Heading should be **Build Networking Recipe**.

Reply:
Done